src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -280,8 +280,26 @@ public override void VisitMethod(IMethodSymbol symbol)
                     AddSpace();
                 }
 
+                if (ShouldMethodDisplayReadOnly(symbol))
```

Comment:
>ShouldMethodDisplayReadOnly [](start = 20, length = 27)

Is this applicable to a function pointer?

Reply:
No, I meant to remove it.

---
In reply to: [422525960](https://github.com/dotnet/roslyn/pull/44078#discussion_r422525960) [](ancestors = 422525960)